tests/console/gdb.pm

```diff
@@ -36,10 +36,16 @@ sub wait_serial_or_die {
 
 
 sub run {
-    my $self = shift;
-    $self->select_serial_terminal;
+    my $self      = shift;
+    my $test_deps = 'gcc glibc-devel gdb';
 
-    zypper_call('in gcc glibc-devel gdb');    #Install test dependencies.
+    $self->select_serial_terminal;
+    # *pidof* binary is normally found in *procps* rpm
+    # except of sle, where it is provided by *sysvinit-tools* rpm
+    # since sle(15-SP3+) *sysvinit-tools* is not preinstalled on JeOS
+    # as systemd's dependency with *sysvinit-tools* was dropped
+    $test_deps .= ' sysvinit-tools' if is_jeos;
```

Comment:
```suggestion
    $test_deps .= ' sysvinit-tools' if is_sle("<16");
```

or something like that. On TW it's not needed and it's also needed on non-JeOS SLE.

Reply:
Yeah, `is_jeos` is not enough. It does not seem to be a problem for [for yast installations](https://openqa.suse.de/tests/4866330#step/gdb/45)

Reply:
Probably pulled in by some other package, but that might be more or less by accident as well.
I don't think special casing JeOS makes much sense here.

Reply:
Yup, it could be just coincidence.